sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/TypeAliases.kt

```diff
@@ -0,0 +1,11 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.Breadcrumb
+import io.sentry.Scope
+import io.sentry.SentryLevel
+import io.sentry.protocol.User
+
+typealias AndroidSentryLevel = SentryLevel
+typealias AndroidSentryUser = User
+typealias AndroidSentryScope = Scope
+typealias AndroidSentryBreadcrumb = Breadcrumb
```

Comment:
Why do we need `typealias`?

Reply:
This makes it easier to work in classes with different types from KMP and Android for example so you immediately know which type you are dealing with. I have been importing those with `as [name]` alot and wanted some consistency. from `SentryLevel and SentryLevel` to `SentryLevel and AndroidSentryLevel`. But it may be good to omit the `Sentry` in the name though